### DISCUSSION TITLE
Bs Swiper one card

### DISCUSSION THREAD

**Discussion by ersingo** [bootscore/bootscore#613] [Q&A]
bs-swiper-card shows items in 4 (xxl), 3 (lg), 2 (md) and 1 (sm) column cards.
bs-swiper-card-autoplay shows items in 4 (xxl), 3 (lg), 2 (md) and 1 (sm) column cards with autoplay.
bs-swiper-card-product shows WooCommerce products in 4 (xxl), 3 (lg), 2 (md) and 1 (sm) column cards.

https://demosite.gen.tr/ana-sayfa/

Hi Basti, I'm experimenting on this page. how can I make it show a single card on all screens?

**Comment by crftwrk** [comment]
Hi Ersin,

the simplest way is to use the `bs-swiper-hero` template, because this has only 1 slide per view on all screen sizes. Override template in your child as described here https://bootscore.me/documentation/bs-swiper/#Overriding_templates_via_theme.

Then locate the `swiper-slide` div and replace content with the card ones.

There are several ways more to achive this, but using an existing template is the best method.

 Solved?
